test/pgtest/copy-from.pt

```diff
@@ -0,0 +1,253 @@
+send
```

Comment:
If this test fails, the table is left behind, so the next execution will also fail, but will drop the table.

Reply:
you can consider using DROP TABLE IF EXISTS here, and expect success as a result.

Reply:
oh, I tried `DROP TABLE <table> IF EXISTS` instead of `DROP TABLE IF EXISTS <table>`. Thanks!